nBTC/sources/nbtc.move

```diff
@@ -276,6 +295,13 @@ public fun get_mint_fee(contract: &NbtcContract): u64 {
     contract.mint_fee
 }
 
+public fun reserves(contract: &NbtcContract): &VecMap<vector<u8>, u64> {
```

Comment:
Let's return sum here

```suggestion
public fun total_deposit(contract: &NbtcContract): u64 {
```

Reply:
I think better we return total depsit for each endpoint